### ISSUE TITLE
Typescript error when trying to use renderThumbInner

### ISSUE THREAD

**Issue by rodnolan** [xiaolin/react-image-gallery#457]
My image items:
```javascript
const allItems =
    product &&
    product.images &&
    product.images.map((image) => {
      return {
          original: image.thumbnailurls[5].url,
          thumbnail: image.thumbnailurls[1].url,
        };
      }
    });
``` 
`product` is a property that is passed into the parent component.

My custom renderer function:
```javascript
const renderThumbInner = (item: ReactImageGalleryItem): React.ReactNode => {
  return (
      <img src={item.thumbnail} />
  );
};
```

Here's my component instance:
```javascript
<ImageGallery
        items={allItems}
        infinite
        showPlayButton={false}
        showNav
        thumbnailPosition="bottom"
        renderThumbInner={renderThumbInner}
      />
```
Here is the error that I get:
```console
No overload matches this call.
  Overload 1 of 2, '(props: Readonly<ReactImageGalleryProps>): ReactImageGallery', gave the following error.
    Type '{ items: { original: string; thumbnail: string; }[]; infinite: true; showPlayButton: false; showNav: true; thumbnailPosition: "bottom"; renderThumbInner: (item: ReactImageGalleryItem) => ReactNode; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactImageGallery> & Readonly<ReactImageGalleryProps> & Readonly<{ children?: ReactNode; }>'.
      Property 'renderThumbInner' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactImageGallery> & Readonly<ReactImageGalleryProps> & Readonly<{ children?: ReactNode; }>'.
  Overload 2 of 2, '(props: ReactImageGalleryProps, context?: any): ReactImageGallery', gave the following error.
    Type '{ items: { original: string; thumbnail: string; }[]; infinite: true; showPlayButton: false; showNav: true; thumbnailPosition: "bottom"; renderThumbInner: (item: ReactImageGalleryItem) => ReactNode; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactImageGallery> & Readonly<ReactImageGalleryProps> & Readonly<{ children?: ReactNode; }>'.
      Property 'renderThumbInner' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<ReactImageGallery> & Readonly<ReactImageGalleryProps> & Readonly<{ children?: ReactNode; }> TS2769'.
```

The docs advise to "see source for reference" and the examples don't cover this scenario so I might be missing something. Has anyone else attempted to use this property? Could you please verify whether my sample code is taking the right approach?

**Comment by xiaolin** [comment]
I recently added the missing `renderThumbInner` to proptypes and default props. (not yet released) https://github.com/xiaolin/react-image-gallery/commit/91feaa003d5adffc506ec85b836e5083f3a79a8a

I think once released it should fix your issue.

**Comment by xiaolin** [comment]
released in version >= v1.0.0
